src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form"
+    >
+      <fieldset class="fieldset">
+        <legend>
+          Personal information
+        </legend>
+        <label class="form-label">
+          Surname:
+          <input
+            type="text"
+            name="surnameField"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-label">
+          Name:
+          <input
+            type="text"
+            name="nameField"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-label">
+          How old are You?
+          <input
+            type="number"
+            name="ageField"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-label">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthDateField"
+          >
+        </label>
+        </label>
+        <label class="form-label">
+          I accept the term of agreement
+          <input
+            type="checkbox"
+            name="agreementField"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          Registration
+        </legend>
+        <label class="form-label">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            required
+            name="emailField"
+          >
+        </label>
+        <label class="form-label">
+          Password:
+          <input
+            type="password"
+            minlength="8"
+            maxlength="20"
+            required
+            name="passwordField"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          An intresting fact about you!
+        </legend>
+        <label class="form-label">
+          Do you love cats?
+          <input
+            type="radio"
+            name="catsField"
+            id="answer1"
```

Comment:
Don't use numbers in classes or ids, it's not clear what it mean

Reply:
thanks for tips, next time i won't use numbers